pkg/proc/proc_test.go

```diff
@@ -2436,13 +2436,24 @@ func TestStacktraceWithBarriers(t *testing.T) {
 	// stack frame with the address of runtime.stackBarrier.
 	// The original return address is saved into the stkbar slice inside the G
 	// struct.
+
+	// In Go 1.8 stack barriers are not inserted by default, this enables them.
```

Comment:
Can you add a `TODO` for ignoring this test on 1.9 when it is no longer applicable.